website/docs/docs/core/about-the-cli.md

```diff
@@ -1,22 +1,25 @@
 ---
-title: "About the CLI"
+title: "About dbt Core"
 id: "about-the-cli"
-sidebar_label: "About the CLI"
+sidebar_label: "About dbt Core"
 ---
 
-dbt ships with a command line interface (CLI) for running your dbt project. This way of running dbt and a dbt project is free and open source.
+[dbt Core](https://github.com/dbt-labs/dbt-core) is an open sourced project where you can develop from the command line and run your dbt project.
+
+To use dbt Core, your workflow generally looks like:
 
-To use the CLI, your workflow generally looks like:
 1. **Build your dbt project in a code editor &mdash;** popular choices include VSCode and Atom.
 
-1. **Run your project from the command line &mdash;** macOS ships with a default Terminal program, however you can also use iTerm or the command line prompt within a code editor to execute dbt commands.
+2. **Run your project from the command line &mdash;** macOS ships with a default Terminal program, however you can also use iTerm or the command line prompt within a code editor to execute dbt commands.
 
 :::info How we set up our computers for working on dbt projects
 
-We've written a [guide](https://discourse.getdbt.com/t/how-we-set-up-our-computers-for-working-on-dbt-projects/243) for our recommended setup when running dbt projects using the CLI.
+We've written a [guide](https://discourse.getdbt.com/t/how-we-set-up-our-computers-for-working-on-dbt-projects/243) for our recommended setup when running dbt projects using dbt Core.
 
 :::
 
-If you're using the CLI, we recommend learning some basics of your terminal to help you work more effectively. In particular, it's important to understand `cd`, `ls` and `pwd` to be able to navigate through the directory structure of your computer easily.
+If you're using the command line, we recommend learning some basics of your terminal to help you work more effectively. In particular, it's important to understand `cd`, `ls` and `pwd` to be able to navigate through the directory structure of your computer easily.
+
+You can find more information on installing and setting up the dbt Core [here](/docs/core/installation).
 
-You can find more information on installing and setting up the dbt CLI [here](/dbt-cli/cli-overview).
+dbt also supports a dbt Cloud CLI and dbt Core, both command line tools that enable you to run dbt commands. The key distinction is the dbt Cloud CLI is tailored for dbt Cloud's infrastructure and integrates with all its [features](/docs/cloud/about-cloud/dbt-cloud-features).
```

Comment:
Is another possible distinction that Core is "build your own?" This might help people understand the diffs.